about/get-involved.php

```diff
@@ -0,0 +1,123 @@
+<?php
+  require_once('includes/template.php');
+  
+  // Required
+  head([
+    'title' =>'Get Involved | Keyman',
+    'css' => ['template.css'],
+    'showMenu' => true
+  ]);           
+?>
+<h2 class="red underline">Get Involved</h2>
+<p>
+  Keyman is a free and open source software project. Continued development of Keyman depends on the generosity of 
+  you, our users. There are many ways in which you can get involved; keep reading to find the best way you can 
+  be a part of the journey:
+</p>
+
+<ul>
+  <li><a href='#donate'>Make a financial donation</a></li>
+  <li><a href='#create-keyboard'>Create a keyboard layout for your language</a>
+  <li><a href='#support-users'>Support other users</a>
+  <li><a href='#test-beta'>Test a new version</a>
+  <li><a href='#write-code'>Write some code</a>
+  <li><a href='#improve-docs'>Improve the documentation</a>
+  <li><a href='#join-team'>Join the team!</a>
+</ul>
+
+<h2 id='donate' class="red">Make a financial donation</h2>
+
+<p>
+  <a href='/donate'>Click here</a> to make a donation to the Keyman project by credit card, 
+  check, or electronic transfer. Your donation will be used to help cover the costs of developing Keyman, including 
```

Comment:
Just another minor wording, you could use "bank transfer" in place of "electronic transfer".  I see the phrase "bank transfer" appear on the other sites having a donate page.